apps/web/src/server/views/applications/case-documentation/documentation-folder.njk

```diff
@@ -37,13 +50,18 @@
 		</div>
 	{% endif %}
 
+	<h2 class="govuk-heading-m">Documents</h2>
 	<!-- TODO: loop to display files in this folder -->
-	{% for documentationFile in documentationFiles %}
-		<div class="govuk-grid-row govuk-grid-column-full">
-			<a class="govuk-link govuk-body-s govuk-!-font-weight-bold" href={{ documentationFile.url }}>
-				{{ documentationFile.fileName }}
-			</a>
-		</div>
-	{% endfor %}
+	{% if documentationFiles |length %}
```

Comment:
It makes sense to use a list here:

{% if documentationFiles |length %}
<ul>
  {% for documentationFile in documentationFiles %}
   <li class="govuk-grid-row govuk-grid-column-full">
	<a class="govuk-link govuk-body-s govuk-!-font-weight-bold" href={{ documentationFile.url }}>
		{{ documentationFile.fileName }}
	</a>
    </li>
    {% endfor %}
</ul>
{% else %}